_data/site.yaml

```diff
@@ -80,24 +88,24 @@ catalog_filters:
 resource_quick_links:
   - title: Data Management & Governance
     url: https://resources.data.gov/categories/data-management-governance/
-  - title: Guidance
-    url: https://resources.data.gov/categories/guidance/
   - title: Data Tools
     url: https://resources.data.gov/categories/data-tools/
-  - title: Case Studies & Examples
-    url: https://resources.data.gov/categories/case-studies-examples/
   - title: Data Inclubator
     url: https://resources.data.gov/categories/data-incubator/
+  - title: Guidance
+    url: https://resources.data.gov/categories/guidance/
+  - title: Case Studies & Examples
+    url: https://resources.data.gov/categories/case-studies-examples/
   - title: Skills Development
     url: https://resources.data.gov/categories/skills-development/
```

Comment:
Reordering for better flow of readability?  Order of introduction of material?  Or some other reason?

Reply:
I figured out how to order the items as expected with flexbox.